incubator/vault/templates/deployment.yaml

```
@@ -118,6 +118,9 @@ spec:
             scheme: {{ if .Values.vault.config.listener.tcp.tls_disable -}}HTTP{{- else -}}HTTPS{{- end }}
           initialDelaySeconds: {{ .Values.vault.liveness.initialDelaySeconds }}
           periodSeconds: {{ .Values.vault.liveness.periodSeconds }}
+          failureThreshold: {{ .Values.vault.liveness.failureThreshold }}
+          successThreshold: {{ .Values.vault.liveness.successThreshold }}
+          timeoutSeconds: {{ .Values.vault.liveness.timeoutSeconds }}
```

Similarly to the below:
```suggestion
          failureThreshold: {{ .Values.vault.liveness.failureThreshold | default 3 }}
          successThreshold: {{ .Values.vault.liveness.successThreshold | default 1 }}
          timeoutSeconds: {{ .Values.vault.liveness.timeoutSeconds | default 1 }}
```